spec/tapioca/dsl/compilers/frozen_record_spec.rb

```diff
@@ -97,6 +97,111 @@ def last_name?; end
               assert_equal(expected, rbi_for(:Student))
             end
 
+            it "can handle annotated fields" do
+              add_ruby_file("student.rb", <<~RUBY)
+                # typed: strong
+
+                class Student < FrozenRecord::Base
+                  extend(T::Sig)
+
+                  self.base_path = __dir__
+
+                  sig { returns(String) }
+                  def first_name
+                    super
+                  end
+
+                  sig { returns(String) }
+                  def last_name
+                    super
+                  end
+
+                  sig { returns(String) }
+                  def location
+                    super
+                  end
```

Comment:
Defining

```ruby
sig { returns(String) }
attr_reader :first_name
```

generates a runtime error in Sorbet when accessed via `Student.first.first_name` since Sorbet thinks this is nilable. Accessing the value via `Student.first["first_name"]` works fine, but I expect most people are using dot notation.

The approach of explicitly defining the method seems like a reasonable tradeoff for type safety.